### PR TITLE
Register aswincreates.is-a.dev

### PR DESCRIPTION
**Pull request by AswinCreates**
Register aswincreates.is-a.dev with TXT record pointing to dh=0613b355bada9aa6ed290515e3510ccfb7706602.

### CODE REVIEW

File: domains/aswincreates.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "AswinCreates",
+           "email": "aswinrout94281@gmail.com",
+           "discord": "952073733705724026"
+        },
+    
+        "record": {
+            "TXT": "dh=0613b355bada9aa6ed290515e3510ccfb7706602"
```
Comment (CuteDog5695) [inline domains/aswincreates.json:9]: This isn't supposed to be for the main domain, please use either A, CNAME or URL records to point to your website

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/aswincreates.json (hunk 1 above)
Comment (AswinCreates) [reply domains/aswincreates.json:9]: i followed all steps that is shown in docs :/

Comment (CuteDog5695) [reply domains/aswincreates.json:9]: > i followed all steps that is shown in docs :/

You made 2 files exactly like this. Main domain shouldn't have the discord txt verification, please point it to your website

Comment (AswinCreates) [reply domains/aswincreates.json:9]: alright, what if i delete all files and recreate? will it work??

Comment (CuteDog5695) [reply domains/aswincreates.json:9]: > alright, what if i delete all files and recreate? will it work??

You know you can just replace the records of the file I requested changes on, right? You can also read the docs on what records we support

Comment (AswinCreates) [reply domains/aswincreates.json:9]: alr

**Pull request closed without merging**